docs/OnnxReleases.md

```diff
@@ -1,53 +1,43 @@
-The ONNX project, going forward, will plan to release roughly on a two month cadence. We follow the [Semver](https://semver.org/) versioning approach and will make decisions as a community on a release by release basis on whether to do a major or minor release.
+The ONNX project, going forward, will plan to release roughly on a four month cadence. We follow the [Semver](https://semver.org/) versioning approach and will make decisions as a community on a release by release basis on whether to do a major or minor release.
 
-Next expected release: May 15th 
+Next expected release: Sept 15th 2020
 
-Release Checklist: 
+The following is the release process for v1.7.0, including a few manual steps that could be automated for next release:
 
 * Install Twine, a utility tool to interact with pypi. Do  - ``pip install twine``
 * Get hold of the username and password for the ‘onnx’ pypi account.
-* Pick a release tag for the new release through mutual consent – Gitter Room for Releases (https://gitter.im/onnx/Releases)
-* Prepare a change log for the release – 
-    * ``git log --pretty=format:"%h - %s" <tag of the previous release>...<new tag>``
-    * And draft a new release statement - https://github.com/onnx/onnx/releases listing out the new features and bug fixes, and potential changes being introduced in the release.
-* Checkout the release tag in a clean branch on your local repo. Make sure all tests pass on that branch.
-* Make sure the VERSION_NUMBER file has up-to-date version - https://github.com/onnx/onnx/blob/master/VERSION_NUMBER
-* Make sure all operators version are even number, if opset version is odd number then increment it and update all corresponding OPs.
-* Make sure all the git submodules are updated
+* Make sure onnx/wheel_builder is working
+* Pick a version number and release tag for the new release through mutual consent – Gitter Room for Releases (https://gitter.im/onnx/Releases)
+* Create a PR to update VERSION_NUMBER (https://github.com/onnx/onnx/blob/master/VERSION_NUMBER) and versioning doc (https://github.com/onnx/onnx/blob/master/docs/Versioning.md)
```

Comment:
nit: It would be nice if you can add a reference PR for this

Reply:
There is one more step below this one to create a pr in release branch to update version number... Should they be combined?

Reply:
These are two different PRs for different purposes. This one is to set the new version number in the master branch. Later one is to reset version number in the release branch after verification.